portfolio/portfolio.go

```diff
@@ -200,43 +201,41 @@ func (p *Base) RemoveAddress(address, description string, coinType currency.Code
 }
 
 // UpdatePortfolio adds to the portfolio addresses by coin type
-func (p *Base) UpdatePortfolio(addresses []string, coinType currency.Code) bool {
+func (p *Base) UpdatePortfolio(addresses []string, coinType currency.Code) error {
 	if strings.Contains(strings.Join(addresses, ","), PortfolioAddressExchange) ||
 		strings.Contains(strings.Join(addresses, ","), PortfolioAddressPersonal) {
-		return true
+		log.Debugln(log.PortfolioMgr, "porfolio already updated")
```

Comment:
Can drop this message as these are for exchange balances (updated via a diff routine) and static balances which don't require updating